### PR TITLE
Allow moderators to see who deleted a message

### LINKED ISSUE #2867: Show which mod deleted message
**Describe your issue**

**Screenshots**

**OS and Chatterino Version**

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Adds support for pubsub deletion messages. Closes #2867.

### CODE REVIEW

File: src/providers/twitch/TwitchMessageBuilder.cpp [hunk 1]
```diff
@@ -1388,4 +1390,40 @@ void TwitchMessageBuilder::deletionMessage(const MessagePtr originalMessage,
...
+        "deleted message from", MessageElementFlag::Text, MessageColor::System);
+    builder
+        ->emplace<TextElement>(action.target.name, MessageElementFlag::Username,
+                               MessageColor::System, FontStyle::ChatMediumBold)
+        ->setLink({Link::UserInfo, action.target.name});
+    builder->emplace<TextElement>("saying:", MessageElementFlag::Text,
+                                  MessageColor::System);
+    if (action.messageText.length() > 50)
+    {
+        builder->emplace<TextElement>(action.messageText.left(50) + "...",
```
Comment (zneix) [inline src/providers/twitch/TwitchMessageBuilder.cpp:1418]: We could make use of unicode character (`…`) in here instead of three dots.

Comment (Mm2PL) [reply src/providers/twitch/TwitchMessageBuilder.cpp]: Resolved in 10645ba

**Comment by Mm2PL** [comment]
Minor bug. I am pretty sure this is not in my code. Probably relates to #2804.

**Comment by Mm2PL** [comment]
@pajlada PRogramming content Okayge? Also I can't request review directly

**Review by pajlada — APPROVED** [review]
:+1:

**Pull request merged**